tests/kontonummer.js

```diff
@@ -116,11 +117,39 @@ describe('Kontonummer suite', () => {
         bank_name: 'Citibank',
         clearing_number: '9044',
         account_number: '12345',
+        warnings: [],
       };
       const result = kontonummer._validateChecksum(mockBank, fakeAccountNumber);
 
       chai.assert.deepEqual(expected, result);
     });
+
+    it('should return object with warning bad_checksum for Swedbank', () => {
+      const mockBankSwedbank = {
+        name    : 'Swedbank',
+        regex   : /^(8[0-9]{4})/,
+        modulo  : 10,
+        lengths : {
+            clearing : 5,
+            account  : 10,
+            control  : 10
+        },
+        zerofill: true,
+        warnOnBadChecksum: true,
+    };
```

Comment:
Incorrect indentation, should be one step more:

```suggestion
      };
```